scripts/files/tests/fs_test.py

```diff
@@ -56,3 +65,20 @@ def test_write_sidecars_key_not_found_s3(capsys: CaptureFixture[str]) -> None:
     # capsys.readouterr().out json string format is not valid which implies
     # we can't parse it to find the actual `msg`
     assert "No sidecar file found; skipping" in capsys.readouterr().out
+
+
+def test_write_sidecars_one_found(capsys: CaptureFixture[str]) -> None:
+    target = mkdtemp()
+    # Add a file to read
+    content = b"test content"
+    path = os.path.join(target, "test.tfw")
+    write(path, content)
+    path_unexisting = os.path.join(target, "test.prj")
```

Comment:
"unexisting" should be "non-existing":

```suggestion
    non_existing_path = os.path.join(target, "test.prj")
```